path/src/browser.ps1

```
@@ -27,8 +59,20 @@ function script:Test-Browser {
 	}
 
 	if (Get-Browser) {
-		New-Item -Path "$FOUNT_DIR/data/installer" -ItemType Directory -Force | Out-Null
-		Set-Content "$FOUNT_DIR/data/installer/auto_installed_chrome" '1'
 		RefreshPath
+		try {
+			New-Item -Path "$FOUNT_DIR/data/installer" -ItemType Directory -Force | Out-Null
+			Set-Content "$FOUNT_DIR/data/installer/auto_installed_chrome" '1'
+		} catch { <# ignore #> }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)